api/app/src/main/kotlin/packit/model/dto/UpdateRolePermission.kt

```diff
@@ -0,0 +1,20 @@
+package packit.model.dto
+
+import org.jetbrains.annotations.NotNull
+
+data class UpdateRolePermission(
+    @field:NotNull
+    val permission: String,
+    val packetId: String? = null,
+    val tagId: Int? = null,
+    val packetGroupId: Int? = null
+)
```

Comment:
I think for the create the FE most likely wont pass any permissions.. And here update is where it can pass in scoped if needed...The idea was it would be weird adding scoped permissions when creating a role as well... But adding a scoped permission when updating makes sense also....

Reply:
OK, but you still want to allow global permissions on create to make life easier for the admin cli/setup scripts? 
I'm not really convinced it's a useful distinction. Setup *might* want to make roles with scoped permissions in some cases for example. I think it would be fine to make role create take no perms and require that to be done in a second request.